R/model_LaTeX.R

```diff
@@ -53,19 +67,19 @@ SEIR_LaTeX <- function(mu) {
   switch(mu + 1,
     helpText(
       r"(Susceptible $$\frac{dS}{dt} = - \beta \frac{ S I}{N^q}$$)",
-      r"(Exposed $$\frac{dE}{dt} = \beta \frac{ S I}{N^q} - \gamma E $$)",
-      r"(Infectious $$\frac{dI}{dt} = \gamma E - \sigma I $$)",
-      r"(Recovered $$\frac{dR}{dt} = \sigma I $$)",
+      r"(Exposed $$\frac{dE}{dt} = \beta \frac{ S I}{N^q} - \gamma {E} $$)",
+      r"(Infectious $$\frac{dI}{dt} = \gamma {E} - \sigma {I} $$)",
+      r"(Recovered $$\frac{dR}{dt} = \sigma {I} $$)",
       r"(Reproductive ratio $$R_0 =  \frac{\beta}{\gamma}$$)",
-      "q-Value", br(), r"($$ 1, frequency-dependent $$)", br(), r"($$ 0, density-dependent $$)"
+      "q-Value $$\\begin{equation} x = \\begin{cases} 1 & \\text{, } frequency − dependent \\\\ 0 & \\text{, } density − dependent \\end{cases}\\end{equation} $$"
     ),
     helpText(
       r"(Susceptible $$\frac{dS}{dt} =\mu_B N - \mu_D S - \beta \frac{ S I}{N^q}$$)",
-      r"(Exposed $$\frac{dE}{dt} = \beta \frac{ S I}{N^q} - \gamma E - \mu_D E$$)",
+      r"(Exposed $$\frac{dE}{dt} = \beta \frac{ S I}{N^q} - \gamma {E} - \mu_D {E}$$)",
       r"(Infectious $$\frac{dI}{dt} = \gamma E - \sigma I - \mu_D I $$)",
       r"(Recovered $$\frac{dR}{dt} = \sigma I - \mu_D R $$)",
       r"(Reproductive ratio $$R_0 =  \frac{\beta}{\gamma}$$)",
-      "q-Value", br(), r"($$ 1, frequency-dependent $$)", br(), r"($$ 0, density-dependent $$)"
+      "q-Value $$\\begin{equation} x = \\begin{cases} 1 & \\text{, } frequency − dependent \\\\ 0 & \\text{, } density − dependent \\end{cases}\\end{equation} $$"
```

Comment:
Please use raw strings when writing $\LaTeX{}$. Inside the raw string you will only have to write $\LaTeX{}$, and won't need to escape backslash characters which are used to call macros and commands.